resources/js/Pages/City/Index.vue

```diff
@@ -58,6 +65,22 @@ function setRating(starIndex) {
   opinionForm.rating = starIndex
 }
 
+function toggleRegulations() {
+  regulationsOpen.value = !regulationsOpen.value
+  
+}
+
+function fetchRegulations() {
+  axios.get('/api/rules/'+props.city.country.name+'/'+props.city.name)
+    .then(response => {
+      rules.pl= response.data.rulesPL
+      rules.en = response.data.rulesEN
+    })
+    .catch(error => {
+      console.log(error)
```

Comment:
We can display error message using toast or something else.

Reply:
@vojcc it's done